ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -18,6 +18,8 @@ aggregation:
   min: "Minimum value (e.g. Min_Ventilation_Air_Flow_Setpoint)"
   total: "Sum total of some set of values (e.g. total_request_heating_count)"
 component:
+  tank : "Storage device for media."
+  lighting : "artificial light source devices or natural illumination by capturing daylight."
```

Comment:
lighting does not belong here. There will be work on lighting in another PR.

Reply:
Lightning has been removed.

Reply:
Can we know when the lightning PR be merged into master branch? Or should we add it in our branch?

Reply:
adding @pisuke who is working on this part.

Reply:
is this still ongoing?

Reply:
@pisuke are you going to add subfield of "Lighting"? we need this to generate lighting point naming.